[.NET 8] The type or namespace name 'ManualLogSource' could not be found

While following the tutorial of writing a basic plugin and running `dotnet build` with no edits to Plugin.cs I get an error:

```
error CS0246: The type or namespace name 'ManualLogSource' could not be found (are you missing a using directive 
 or an assembly reference?) [C:\Program Files (x86)\Steam\steamapps\common\Timberborn\BepInEx\plugins\MyFirs 
tPlugin\MyFirstPlugin.csproj]
```

.NET version: 8.0.300
Template version: 2.0.0-be.3

## How to reproduce

1. Create a project using this command:
`dotnet new bepinex5plugin -n MyFirstPlugin -T netstandard2.1 -U 2023.1.20`

2. Add referencing game libraries:
`dotnet add package Timberborn.GameLibs --version 0.5.7-r.0`

3. Run `dotnet build`

--- a/BepInEx.Templates/templates/BepInEx5.PluginTemplate/Plugin.cs
+++ b/BepInEx.Templates/templates/BepInEx5.PluginTemplate/Plugin.cs
@@ -1,4 +1,5 @@
 ﻿using BepInEx;
+using BepInEx.Logging;
 
 namespace BepInEx5.PluginTemplate;
 
@@ -10,7 +11,7 @@ public class Plugin : BaseUnityPlugin
     private void Awake()
     {
         // Plugin startup logic
-        this.Logger = Logger;
+        Logger = base.Logger;
         Logger.LogInfo($"Plugin {MyPluginInfo.PLUGIN_GUID} is loaded!");
     }
 }

--- a/BepInEx.Templates/templates/BepInEx6.PluginTemplate.NET.CoreCLR/Plugin.cs
+++ b/BepInEx.Templates/templates/BepInEx6.PluginTemplate.NET.CoreCLR/Plugin.cs
@@ -1,4 +1,5 @@
 ﻿using BepInEx;
+using BepInEx.Logging;
 using BepInEx.NET.Common;
 
 namespace BepInEx6.PluginTemplate.NET.CoreCLR;
@@ -11,7 +12,7 @@ public class Plugin : BasePlugin
     public override void Load()
     {
         // Plugin startup logic
-        this.Log = base.Log;
+        Log = base.Log;
         Log.LogInfo($"Plugin {MyPluginInfo.PLUGIN_GUID} is loaded!");
     }
 }

--- a/BepInEx.Templates/templates/BepInEx6.PluginTemplate.NET.Framework/Plugin.cs
+++ b/BepInEx.Templates/templates/BepInEx6.PluginTemplate.NET.Framework/Plugin.cs
@@ -1,4 +1,5 @@
 ﻿using BepInEx;
+using BepInEx.Logging;
 using BepInEx.NET.Common;
 
 namespace BepInEx6.PluginTemplate.NET.Framework;
@@ -11,7 +12,7 @@ public class Plugin : BasePlugin
     public override void Load()
     {
         // Plugin startup logic
-        this.Log = base.Log;
+        Log = base.Log;
         Log.LogInfo($"Plugin {MyPluginInfo.PLUGIN_GUID} is loaded!");
     }
 }

--- a/BepInEx.Templates/templates/BepInEx6.PluginTemplate.Unity.Il2Cpp/Plugin.cs
+++ b/BepInEx.Templates/templates/BepInEx6.PluginTemplate.Unity.Il2Cpp/Plugin.cs
@@ -1,4 +1,5 @@
 ﻿using BepInEx;
+using BepInEx.Logging;
 using BepInEx.Unity.IL2CPP;
 
 namespace BepInEx6.PluginTemplate.Unity.Il2Cpp;
@@ -11,7 +12,7 @@ public class Plugin : BasePlugin
     public override void Load()
     {
         // Plugin startup logic
-        this.Log = base.Log;
+        Log = base.Log;
         Log.LogInfo($"Plugin {MyPluginInfo.PLUGIN_GUID} is loaded!");
     }
 }

--- a/BepInEx.Templates/templates/BepInEx6.PluginTemplate.Unity.Mono/Plugin.cs
+++ b/BepInEx.Templates/templates/BepInEx6.PluginTemplate.Unity.Mono/Plugin.cs
@@ -1,4 +1,5 @@
 ﻿using BepInEx;
+using BepInEx.Logging;
 using BepInEx.Unity.Mono;
 
 namespace BepInEx6.PluginTemplate.Unity.Mono;
@@ -11,7 +12,7 @@ public class Plugin : BaseUnityPlugin
     private void Awake()
     {
         // Plugin startup logic
-        this.Logger = base.Logger;
+        Logger = base.Logger;
         Logger.LogInfo($"Plugin {MyPluginInfo.PLUGIN_GUID} is loaded!");
     }
 }

